src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ProgressBar.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <v-tooltip :text="`${hoursRemaining} Hours Remaining`" location="top">
```

Comment:
This component looks extremely similar to `WorkExperienceReferenceProgressBar.vue` can we combine?

Reply:
Okay I swapped it out with the progressBar. Nice cleanup.